src/main/java/Duke.java

```diff
@@ -55,6 +71,45 @@ private static void addTask(String description) {
         System.out.println("____________________________________________________________");
     }
 
+    public static void addTodoTask(String description) {
+        if (description.isEmpty()) {
+            System.out.println("Anyhow one...write your task properly leh :(");
+            return;
+        }
+        tasks[taskCount++] = new Todo(description);
+        System.out.println("____________________________________________________________");
```

Comment:
maybe instead of pasting the long line each time, u can save the line in a variable like line and use that instead :)